packages/commcare/test/index.js

```diff
@@ -320,3 +320,44 @@ describe('getCases', () => {
     ]);
   });
 });
+
+describe('createUser', ()=>{
+  it('should create a user', async () => {
+    testServer
+      .intercept({
+        path: `/a/${domain}/api/v0.5/user`,
+        method: 'POST',
+      })
+      .reply(201, () => {
+        // simulate a return from commcare
+        return {
+       
+        id:'123456'
```

Comment:
A minor detail but formatting is a bit quirky here. Do you have `prettier` set up?

Reply:
Yes I have. I have fixed it